tests/components/home_connect/test_init.py

```diff
@@ -294,3 +299,52 @@ async def test_services_exception(
 
     with pytest.raises(ValueError):
         await hass.services.async_call(**service_call)
+
+
+TEST_UNIQUE_ID_PREFIX = "unique_id_prefix_"
+
+
+@pytest.mark.parametrize(
+    ("old_unique_id_suffix", "expected_unique_id_suffix"),
+    (
+        (
+            old_unique_id_suffix,
+            expected_unique_id_suffix,
+        )
+        for old_unique_id_suffix, expected_unique_id_suffix in OLD_NEW_UNIQUE_ID_SUFFIX_MAP.items()
+    ),
+)
```

Comment:
I'd rather use some strings here to keep the tests uncoupled from the code, because if someone now changes the map, the test will still work